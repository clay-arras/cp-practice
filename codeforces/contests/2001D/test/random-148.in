1
19
2 12 2 18 8 16 9 19 25 24 1 18 22 17 16 20 14 17 9
