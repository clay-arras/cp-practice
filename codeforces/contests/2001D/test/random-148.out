14
12 2 18 8 19 25 24 1 22 16 20 14 17 9
